Source/AssetRipper.GUI.Electron/Pages/Assets/FontTabContent.cshtml

```diff
@@ -0,0 +1,94 @@
+﻿@model ValueTuple<byte[], string>
+
+@if (!string.IsNullOrEmpty(Model.Item2))
+{
+    <p>This file type: @Model.Item2</p>
```

Comment:
This can be put in the Information tab. It should be easy to add in the `InformationTabExtensions`. Don't worry about any localizations. I'll handle them all at once.